_posts/2023-12-20-dsaa-wrapped-2023/dsaa-wrapped-2023.Rmd

```diff
@@ -0,0 +1,83 @@
+
+# The learning never stops
+
+At DSAA, the learning never stops!
+
+In our team-wide journal clubs, we learned about:
+
+- [The Five Ideals](https://www.youtube.com/watch?v=xyP3c8VWplM)  
+- [Lawsuits in the domain of generative AI](https://www.theverge.com/2023/1/17/23558516/ai-art-copyright-stable-diffusion-getty-images-lawsuit)  
+- [Problems in the deployment of ML models in healthcare](https://www.theverge.com/2023/1/17/23558516/ai-art-copyright-stable-diffusion-getty-images-lawsuit)  
+- [Clean code in Python](https://github.com/zedr/clean-code-python)  
+- and more!
+
+This year, our Advanced Analytics (AA) sub-team had **more than 10** journal club sessions that dived into the worlds of Natural Language Processing (NLP), performance optimization, and project evaluation. The AA team also had the chance to take their learning outside of the office, by going on a field trip to the Thomas Fisher Rare Book Library at U of T, which had an exhibit of [Data Visualization Throughout History](https://fisher.library.utoronto.ca/exhibition/emerging-patterns-data-visualization-throughout-history).
+
+We were also fortunate to have Posit host a number of workshops for us, covering the topics of [pointblank](https://github.com/rstudio/pointblank) and [MLOps with vetiver](https://juliasilge.github.io/mlops-with-vetiver/#/title-slide).
+
+# Sharing ideas
+
+This year, we didn't just learn new things, our team members also had opportunities to exchange ideas, and share insights and experiences through talks and conferences from MIQI to the Renal Insight User Group.
+
+We also hope to continue to raise our count of published blog posts! This past year, we've had team members share **3 blog posts** about [developing and deploying AI solutions in the hospital](https://lks-chart.github.io/blog/posts/2023-01-27-with-a-little-help-from-my-friends/), [creating reproducible environments for building medical imaging machine learning models with Nix](https://lks-chart.github.io/blog/posts/2023-02-28-medical-imaging-with-nix/), and [discrete event simulation in healthcare](https://lks-chart.github.io/blog/posts/2023-03-17-introduction-to-discrete-event-simulations-in-healthcare/).
+
+# The Party Planning Pandas and beyond! 🎉
+
+Let's conclude our DSAA Wrapped by recounting some of our memorable team moments from this past year, organized by our social committee: The Party Planning Pandas 🐼.
+
+- Nerf Gun Challenge: With our in-office stash of nerf guns, during this challenge, our team learned that precision and luck go hand in hand.  
+- Valentine's Day & Zoom trivia: This year, our data science team embraced AI art generators to craft charming (and maybe a little scary) Valentine's Day cards as well as adorable baby animal versions of our team.  
```

Comment:
Do we have any of these saved? Would be cool to share (if they're not too scary!!)